starters/basic-starter/pages/[...slug].tsx

```diff
@@ -45,11 +44,13 @@ export default function NodePage({ node, menus }: NodePageProps) {
   );
 }
 
-// This fetches paths from Drupal and builds static pages.
+// Use the 'paths' key to specify wanted paths to be pre-rendered at build time.
 // See https://nextjs.org/docs/basic-features/data-fetching/get-static-paths.
 export async function getStaticPaths(context): Promise<GetStaticPathsResult> {
```

Comment:
We could remove the context argument since it's unused.

```suggestion
export async function getStaticPaths(): Promise<GetStaticPathsResult> {
```